src/bowling2.js

```diff
@@ -0,0 +1,52 @@
+function Bowling(){
+  this.rolls = [];
+}
+
+    Bowling.prototype.roll = function(pins){
+          this.rolls.push(pins);
+    };
+
+
+    Bowling.prototype.score = function() {
```

Comment:
This function has too many statements. (18)<br>This function's cyclomatic complexity is too high. (5)